rust-runtime/aws-smithy-http-server-python/src/lambda.rs

```diff
@@ -86,29 +127,49 @@ struct PyConfig {
 #[pyclass(name = "LambdaContext")]
 pub struct PyLambdaContext {
     /// The AWS request ID generated by the Lambda service.
+    ///
+    /// :type str:
     #[pyo3(get)]
     request_id: String,
+
     /// The execution deadline for the current invocation in milliseconds.
+    ///
+    /// :type int:
     #[pyo3(get)]
     deadline: u64,
+
     /// The ARN of the Lambda function being invoked.
+    ///
+    /// :type str:
     #[pyo3(get)]
     invoked_function_arn: String,
+
     /// The X-Ray trace ID for the current invocation.
+    ///
+    /// :type typing.Optional\[str\]:
```

Comment:
Escape chars are required for Rustdoc because the syntax `[foo]` is for [intra-doc linking](https://doc.rust-lang.org/rustdoc/write-documentation/linking-to-items-by-name.html) and Rustdoc tries to link it to element `foo` and fails if it can't. We are [unescaping](https://github.com/awslabs/smithy-rs/blob/5f355d1839dc25d8161a3f1a3689cb46f38d027f/rust-runtime/aws-smithy-http-server-python/examples/stubgen.py#L34-L43) those chars in our stubgen script, but I don't know if there is a way to unescape them from `help()` function too.